Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -394,3 +394,44 @@ label monika_monikamoddev:
     if not _return:
         $ mas_unlockFailedWRS('monika_monikamoddev')
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="monika_twitter",
```

Comment:
Also, one issue. This label already exists in `script-topics.rpy`. You'll want to use a label which doesn't conflict. I recommend making this something like `monika_twiter_react` to differentiate.